draft-ietf-quic-recovery.md

```diff
@@ -1514,6 +1517,7 @@ variables as follows:
    bytes_in_flight = 0
    congestion_recovery_start_time = 0
    ssthresh = infinite
+   first_rtt_sample = never
```

Comment:
We use 0 to initialize the other time variables.

Reply:
Yeah, but that means understanding that 0 is not a valid time, and I don't know if that is true.  But I guess that is the convention, so I'll fix it.